application/cli-app/src/main/java/org/mobilitydata/gtfsvalidator/Main.java

```diff
@@ -139,9 +141,24 @@ public static void main(String[] args) {
                 }
             });
 
+            boolean asProto = false;
```

Comment:
Please declare and affect this variable in a single line, like `zipInputPath` for example. Also finalize other command line variables.

Reply:
Modifying this, but it seems that zipInputPath is the only command line variable that can not be finalize since it is modified in the subsequent lines https://github.com/MobilityData/gtfs-validator/blob/efae2a70e27c6917323a39ac18f9f7ac159fc9bb/application/cli-app/src/main/java/org/mobilitydata/gtfsvalidator/Main.java#L99